src/app/configs/endpoints.ts

```diff
@@ -8,5 +8,6 @@ export const AUTH_ENDPOINTS = {
 };
 
 export const BOOK_ENDPOINTS = {
-  addBook: `${BASE_URL}/${SUFFIX}/book`,
+  addBook: `${BASE_URL}/${SUFFIX}/books`,
+  ListBook: `${BASE_URL}/${SUFFIX}/Listbook`,
```

Comment:
The `ListBook` endpoint does not exist in the back-end. The correct endpoint is `${BASE_URL}/${SUFFIX}/books`
Coding style consistency is also important. Do not capitalize letters that do not require capitalization.